packages/ui/src/components/Button/Button.tsx

```
@@ -0,0 +1,52 @@
+    },
+    ref
+  ) => (
+    <button
+      ref={ref}
+      className={`${buttonRecipe({ size, variant })} ${className}`}
+      {...rest}
+    >
+      {isLoading ? (
+        <>
+          // TODO Spinner 컴포넌트 넣을 예정
+          <span>로딩</span>
```

Biome (1.9.4)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)